Tests/Directory.Build.targets

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <PropertyGroup>
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>latest</LangVersion>
```

Comment:
This should probably be extracted to a small standalone PR for the sake of sane history

Reply:
The change has been extracted into https://github.com/SonarSource/sonar-scanner-msbuild/pull/1793